### PR TITLE
BRC4_genome_loader docs draft

### PR DESCRIPTION
**Pull request by vsitnik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,5 +64,81 @@ cd ./ensembl-genomio
 ./scripts/setup/docs/build_sphinx_docs.sh
 ```
 
+## Pipelines
+
+### Initialising and running eHive-based pipelines
+
+Pipelines are derived from [`Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf`](https://github.com/Ensembl/ensembl-hive/blob/version/2.6/modules/Bio/EnsEMBL/Hive/PipeConfig/HiveGeneric_conf.pm),
+or from [`'Bio::EnsEMBL::Hive::PipeConfig::EnsemblGeneric_conf`](https://github.com/Ensembl/ensembl-hive/blob/version/2.6/modules/Bio/EnsEMBL/Hive/PipeConfig/EnsemblGeneric_conf.pm),
```
Comment (ens-LCampbell) [inline README.md:72]: Single quote on this line without closing quote ( 'Bio...)

File: docs/BRC4_genome_loader.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+## BRC4_genome_loader
+### Module: [[Bio::EnsEMBL::Pipeline::PipeConfig::BRC4_genome_loader_conf](../lib/perl/Bio/EnsEMBL/Pipeline/PipeConfig/BRC4_genome_loader_conf.pm)
```
Comment (ens-LCampbell) [inline docs/BRC4_genome_loader.md:2]: Extra square bracket [ without closing partner on line 2

File: docs/BRC4_genome_loader.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+| `-pipeline_dir` | | directory to store results to
+
+
+
+#### Notes
+2nd mode for adding
+
+
+
+### Input data {#input}
```
Comment (ens-LCampbell) [inline docs/BRC4_genome_loader.md:60]: Whats the purpose of {#input} on here ?

File: README.md [hunk 4]
```diff
@@ -1,17 +1,28 @@
 # ensembl-genomio
 Pipelines to turn basic genomic data into Ensembl cores and back
 
-A set of ehive pipelines:
+This is a mulitlanguage (Perl, Python) repo providing eHive pipelines
+and various scripts (see below) to prepare genomic data and load it as [Ensembl core database](http://www.ensembl.org/info/docs/api/core/index.html) or to dump such core databases as file bundles.
 
-* Genome dumper: dumps flat files from an Ensembl core database.
-* Genome loader: creates an Ensembl core database from a set of flat files.
+The files composing data bundle are of various formats (fasta, gff3, json) and should follow the corresponding [specification](docs/BRC4_genome_loader.md#input).
```
Comment (ens-LCampbell) [inline README.md:7]: "The files composing a genomic data bundle are of various formats (fasta,...."

Suggestion to alter this sentence  on line 7

**Comment by ens-LCampbell** [comment]
@vsitnik Just a couple of typos and some comments/suggestions. Other than that Im happy with this Doc template.

File: README.md (hunk 1 above)
Comment (vsitnik) [reply README.md:72]: thanks!

File: docs/BRC4_genome_loader.md (hunk 3 above)
Comment (vsitnik) [reply docs/BRC4_genome_loader.md:60]: to use as an anchor (doesn't work for github)

**Pull request merged**